airflow/api_fastapi/core_api/routes/public/tasks.py

```diff
@@ -17,19 +17,49 @@
 
 from __future__ import annotations
 
+from operator import attrgetter
+
 from fastapi import HTTPException, Request, status
 
 from airflow.api_fastapi.common.router import AirflowRouter
-from airflow.api_fastapi.common.types import get_class_ref
-from airflow.api_fastapi.core_api.datamodels.tasks import TaskResponse
+from airflow.api_fastapi.core_api.datamodels.tasks import TaskCollectionResponse, TaskResponse
 from airflow.api_fastapi.core_api.openapi.exceptions import create_openapi_http_exception_doc
 from airflow.exceptions import TaskNotFound
 from airflow.models import DAG
-from airflow.models.mappedoperator import MappedOperator
 
 tasks_router = AirflowRouter(tags=["Task"], prefix="/dags/{dag_id}/tasks")
 
 
+@tasks_router.get(
+    "/",
+    responses=create_openapi_http_exception_doc(
+        [
+            status.HTTP_400_BAD_REQUEST,
+            status.HTTP_401_UNAUTHORIZED,
+            status.HTTP_403_FORBIDDEN,
```

Comment:
401 and 403 do not  need to be specified here anymore, because https://github.com/apache/airflow/pull/43932 adds by default 401 and 403 to all routes. (all authenticated routes).

This will automatically be inherited here. We still need to keep others like 400 or 404.

Reply:
Done, removed 401 and 403. Thanks!